tests/contrib/kafka/test_kafka.py

```diff
@@ -154,3 +156,66 @@ def test_analytics_without_rate(producer, consumer):
         message = None
         while message is None:
             message = consumer.poll(1.0)
+
+
+def retry_until_not_none(factory):
+    for i in range(10):
+        x = factory()
+        if x is not None:
+            return x
+        time.sleep(0.1)
+    return None
+
+
+def test_data_streams_kafka():
+    import six
```

Comment:
Do we need to inline these imports? Aren't they already imported at the top of the file?